staging/volumes/azure_disk/claim/blob-based-disk/dedicated-hdd/pod-uses-dedicated-hdd.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: pod-uses-dedicated-hdd-5g
+  labels:
+    name: storage
+spec:
+  containers:
+  - image: nginx
+    name: az-c-01
+    command:
+    - "/bin/sh"
+    - "-c"
+    - while true; do echo $(date) >> /mnt/blobdisk/outfile; sleep 1; done
```

Comment:
seems inconsistent that you quoted "/bin/sh" "-c" and not this line (ditto other files)

Reply:
I have removed all ", they are all unnecessary, thx for the check